dashboards-observability/.cypress/integration/1_event_analytics.spec.js

```diff
@@ -35,13 +35,13 @@ describe('Has working breadcrumbs', () => {
   it('Redirect to correct page on breadcrumb click', () => {
     landOnEventExplorer();
     cy.wait(delay * 3);
-    cy.get('.euiBreadcrumb').contains('Explorer').click();
+    cy.get('.euiBreadcrumb[href="#/event_analytics/explorer"]').contains('Explorer').click();
```

Comment:
Are these cypress changes for this PR?

Reply:
Not for this PR, but they failed when I ran everything so I thought I might as well fix it if it's being flaky